### PR TITLE
Lets review the android tic tac toe pt br 2

### PR DESCRIPTION
**Pull request by caioortiz27**

### CODE REVIEW

File: content/brazilian-portuguese/android-tictactoe/printconsole.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+weight: 4
+---
+Às vezes, o jogo ou aplicativo pode não funcionar da maneira esperada ou você deseja gravar algo no console para fins de registro. Fora da depuração do aplicativo, você pode usar `System.out.println()` para escrever sua mensagem. Procure suas mensagens na janela "logcat" na parte inferior do Android Studio:
+<img src="../resources/_gen/images/println.gif" height="60%" width="60%" title="System.out.println()" alt="Janela Logcat no Android Studio mostrando mensagens criadas usando a função de registro System.out.printLn"/>
+
+{{% notice tip %}}
+
+## Trabalhando juntos
+
+1. Sem remover as aspas, tente alterar `"Reiniciando o jogo Tac-Tac-Toe!"` para `"Reiniciando o jogo Tic-Tac-Toe!"` ou qualquer outra frase que desejar. Pressione executar para ver se alguma coisa muda.
```
Comment (Renato3005) [inline content/brazilian-portuguese/android-tictactoe/printconsole.md:14]: Translate "Tic-Tac-Toe"
@ozhang22 the proposal "try changing `"Restarting the Tac-Tic-Toe Game!"` to `"Restarting the Tic-Tac-Toe Game!"` looks weird, as it changes nothing. Is that right?

File: content/brazilian-portuguese/android-tictactoe/printconsole.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+3. Não exclua chaves `{` ou `}`.
+
+{{% /notice %}}
+
+`System.out.println` é útil quando você deseja corrigir bugs no código, mas seu usuário (a pessoa que usa o aplicativo) não consegue ver o texto impresso com esta função. Mas podemos fazer isso com brindes! Um brinde é um texto pequeno e rápido que tornamos visível para o usuário. Veja como fazer um brinde:
+
+```kotlin
+Toast.makeText(this, "Texto que queremos mostrar", Toast.LENGTH_SHORT).show()
+```
+<img src="../resources/_gen/images/toast_example.png" title="Toast Example" alt="exemplo de torrada. Você pode usar a função brinde.makeText com o contexto dos argumentos, a string da mensagem e Toast.LENGTH_LONG ou Toast.LENGTH_SHORT para exibir uma mensagem de boas-vindas, como Bem-vindo ao Tic-Tac-Toe"/>
```
Comment (Renato3005) [inline content/brazilian-portuguese/android-tictactoe/printconsole.md:37]: translate "Tic-Tac-Toe"

File: content/brazilian-portuguese/android-tictactoe/answer-key.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+### Quebra-cabeça 1
+```xml
+<resources>
+ <string name="app_name">Jogo da Velha</string>
+</resources>
+```
+
+### Quebra-cabeça 2
+```kotlin
+Toast.makeText(this, "Welcome to Tic-Tac-Toe", Toast.LENGTH_LONG).show()
```
Comment (Renato3005) [inline content/brazilian-portuguese/android-tictactoe/answer-key.md:17]: "Welcome to Tic-Tac-Toe" might need translation.

File: content/brazilian-portuguese/android-tictactoe/puzzle1.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 2
+---
+
+## Execute o jogo
+Pressione o botão play (►) na parte superior do Android Studio para iniciar o jogo.
+<img src="../resources/_gen/images/run_app.png" height="30%" width="30%" title="Botão Executar aplicativo" alt="Botão Executar aplicativo"/>
+
+Você verá que o título na parte superior do aplicativo diz Toe-Tac-Tic (isso não parece certo!); vamos mudar para Tic-Tac-Toe.
```
Comment (Renato3005) [inline content/brazilian-portuguese/android-tictactoe/puzzle1.md:12]: @ozhang22 Tic-Tac-Toe translated to Pt-BR is "Jogo da Velha". In this line, I suppose the working application downloaded by the reader already has this mistake so it is supposed to be fixed. Should we proceed by suggesting the student to change "Toe-Tic-Tac" to "Jogo da Velha", to "Tic-Tac-Toe" or we should do something else?

File: content/brazilian-portuguese/android-tictactoe/puzzle1.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+## Resolva o quebra-cabeça
+Clique duas vezes no arquivo `strings.xml` em app &rarr; res &rarr; valores:
+<img src="../resources/_gen/images/open_strings_file.gif" height="30%" width="30%" title="localização Strings.xml" alt="Mostra como navegar no toe strings.xml no Android Studio"/>
+
+Como você pode modificar este arquivo para dizer Tic-Tac-Toe?
```
Comment (Renato3005) [inline content/brazilian-portuguese/android-tictactoe/puzzle1.md:18]: Awaiting instructions to translate or not "Tic-Tac-Toe".

File: content/brazilian-portuguese/android-tictactoe/puzzle2.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Quebra-cabeça 2 - Bem-vindo usuário"
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 5
+---
+Vamos adicionar uma mensagem de boas-vindas que o usuário poderá ver quando iniciar seu aplicativo!
+
+Vá para **MainActivity.kt** e encontre `// PUZZLE 2` no código e escreva um brinde que dá as boas-vindas ao usuário ao seu jogo com "Bem-vindo ao Tic-Tac-Toe!". Agora pressione o botão play (► ) na parte superior do Android Studio e seu aplicativo deverá ficar assim:
```
Comment (Renato3005) [inline content/brazilian-portuguese/android-tictactoe/puzzle2.md:9]: Awaiting instructions to translate or not "Tic-Tac-Toe".

File: content/brazilian-portuguese/android-tictactoe/puzzle2.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+<img src="../resources/_gen/images/toast_success.png" height="35%" width="27%" title="Brinde de sucesso" alt="Exemplo de jogo com texto de boas-vindas bem-sucedido ao Tic Tac Toe exibido usando brinde"/>
```
Comment (Renato3005) [inline content/brazilian-portuguese/android-tictactoe/puzzle2.md:11]: Awaiting instructions to translate or not "Tic-Tac-Toe".

**Review by Renato3005 — CHANGES_REQUESTED** [review]
All reviewd. There might be a problem with the `english` coding (`printconsole.md`, line 14). It is needed to reconsider if "Tic-Tac-Toe" must be translated or not (details in `puzzle1.md`, line 12).

File: content/brazilian-portuguese/android-tictactoe/_index.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+---
+
+## Comece sua Aventura de Jogo Android
+
+Você já se perguntou como poderia construir seu próprio Jogo da Velha?
+
+Você está com sorte, hoje é o dia em que você poderá construir o seu próprio!
+
+
+<img src="resources/_gen/images/game_play.gif" height="30%" width="22%" title="Android Tic-Tac-Toe Game" alt="Android Tic-Tac-Toe Game"/>
```
[the remark this answers is not in the document]
Comment (caioortiz27) [reply content/brazilian-portuguese/android-tictactoe/_index.md:19]: done

File: content/brazilian-portuguese/android-tictactoe/_index.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+## Começando
+
+## O que é Android?
+O Android é atualmente o sistema operacional móvel mais usado no mundo! O Android Studio permite que você crie aplicativos que podem ser usados por esses dispositivos Android. Na verdade, se você tiver um telefone Android, poderá até baixar o aplicativo que está criando hoje em seu telefone!
```
[the remark this answers is not in the document]
Comment (caioortiz27) [reply content/brazilian-portuguese/android-tictactoe/_index.md:24]: done

File: content/brazilian-portuguese/android-tictactoe/_index.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+## Sobre esse projeto
+
+Neste projeto, fornecemos a você um aplicativo Android quase funcional que é um jogo de Jogo da Velha. Ele precisa de sua ajuda para corrigir alguns problemas e adicionar algumas partes para torná-lo um ótimo jogo!
```
[the remark this answers is not in the document]
Comment (caioortiz27) [reply content/brazilian-portuguese/android-tictactoe/_index.md:28]: done

File: content/brazilian-portuguese/android-tictactoe/android-setup.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
+---
+title: "Configuração do Android"
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 2
+---
+Siga as etapas abaixo para instalar o Android Studio em seu computador. Se você já tiver o Android Studio instalado, abra-o e pule para a próxima etapa!
+
+Essas etapas têm exemplos para instalação em computadores Windows. No entanto, as etapas são semelhantes para outros tipos de computadores também.
```
[the remark this answers is not in the document]
Comment (caioortiz27) [reply content/brazilian-portuguese/android-tictactoe/android-setup.md:9]: done

File: content/brazilian-portuguese/android-tictactoe/android-setup.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+{{% /notice %}}
+
+## Abra os arquivos do projeto
+1. Clique em `Abrir um projeto existente` na tela inicial do Android Studio.
+2. Procure pelo local onde você baixou os arquivos do projeto.
+3. Selecione a pasta 'TicTacToe' dentro da pasta 'TicTacToe' que você descompactou anteriormente.
+ - A pasta 'TicTacToe' a ser selecionada deve ter um ícone verde do Android ao lado, indicando que é uma pasta de projeto Android.
+<img src="../resources/_gen/images/open_android_project.gif" height="40%" width="40%" title="Abrir projeto TicTacToe" alt="Mostra como abrir o projeto TicTacToe no Android Studio"/>
+
+Execute o Android Studio e abra o projeto. Sua visão será semelhante a esta:
```
[the remark this answers is not in the document]
Comment (caioortiz27) [reply content/brazilian-portuguese/android-tictactoe/android-setup.md:58]: done

File: content/brazilian-portuguese/android-tictactoe/answer-key.md (hunk 3 above)
Comment (caioortiz27) [reply content/brazilian-portuguese/android-tictactoe/answer-key.md:17]: done

File: content/brazilian-portuguese/android-tictactoe/puzzle1.md (hunk 4 above)
Comment (caioortiz27) [reply content/brazilian-portuguese/android-tictactoe/puzzle1.md:12]: N/A

File: content/brazilian-portuguese/android-tictactoe/puzzle1.md (hunk 5 above)
Comment (caioortiz27) [reply content/brazilian-portuguese/android-tictactoe/puzzle1.md:18]: done

File: content/brazilian-portuguese/android-tictactoe/android-setup.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+<img src="../resources/_gen/images/android_studio.png" height="60%" width="60%" title="IDE do Android Studio" alt="Exemplo da IDE do Android Studio"/>
+
+## Aceite as Licenças
+Antes de poder executar o código, você precisará aceitar as licenças do Android seguindo as etapas abaixo:
+1. Clique no botão `Terminal` na parte inferior da janela do Android Studio.
+2. Digite o comando abaixo, substituindo '<usuário>' pelo nome de usuário do seu computador.
```
Comment (ozhang22) [inline content/brazilian-portuguese/android-tictactoe/android-setup.md:64]:
```suggestion
2. Digite o comando abaixo, substituindo '<USER_NAME>' pelo nome de usuário do seu computador.
```

File: content/brazilian-portuguese/android-tictactoe/booleans.md [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+
+- 54 < (10 + 32)
+- (37 / 5) == 7
+- "Hello" + "World" == "Hello World"
+- false == false
+{{% /notice %}}
+
+## Operadores Booleanos
+
+Você também pode conectar expressões booleanas usando o operador `&&` (E) e o operador `||` (OU). Por exemplo, suponha que eu pergunte: "Você é humano e Nuvi é um robô?" A palavra "e" conecta as duas perguntas verdadeiro-falso juntas. Neste caso, como é verdade que você é humano e também é verdade que Nuvi é um robô, então o resultado geral é `verdadeiro`. Aqui está um gráfico que descreve o que acontece quando conectamos booleanos juntos:
```
Comment (ozhang22) [inline content/brazilian-portuguese/android-tictactoe/booleans.md:34]:
```suggestion
Você também pode conectar expressões booleanas usando o operador `&&` (E) e o operador `||` (OU). Por exemplo, suponha que eu pergunte: "Você é humano e Nuvi é um robô?" A palavra "e" conecta as duas perguntas verdadeiro-falso juntas. Neste caso, como é verdade que você é humano e também é verdade que Nuvi é um robô, então o resultado geral é `true`. Aqui está um gráfico que descreve o que acontece quando conectamos booleanos juntos:
```

File: content/brazilian-portuguese/android-tictactoe/conditionals.md [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+if (isRainingOutside == true) {
+    takeUmbrella = true;
+} else {
+    takeUmbrella = false;
+}
+```
+
+Para o nosso jogo, devemos usar instruções `if` para determinar se um jogador tem 3 correspondências verticais, horizontais ou diagonais.
+
+Dentro do `()` ao lado de `if`, você deve especificar uma _expressão booleana_. **Booleans** são declarações `verdadeiras` ou `falsas`. Podemos utilizar isso em nossa instrução `if`. Se a expressão for `verdadeira`, o código dentro de `{ }` é executado. Se a expressão for `falsa`, o código dentro de `{ }` após a instrução `else` é executado.
```
Comment (ozhang22) [inline content/brazilian-portuguese/android-tictactoe/conditionals.md:21]:
```suggestion
Dentro do `()` ao lado de `if`, você deve especificar uma _expressão booleana_. **Booleans** são declarações `true` ou `false`. Podemos utilizar isso em nossa instrução `if`. Se a expressão for `verdadeira`, o código dentro de `{ }` é executado. Se a expressão for `falsa`, o código dentro de `{ }` após a instrução `else` é executado.
```

File: content/brazilian-portuguese/android-tictactoe/puzzle1.md (hunk 4 above)
Comment (ozhang22) [reply content/brazilian-portuguese/android-tictactoe/puzzle1.md]: I think this is OK, let's have the student change Toe-Tic-Tac to Jogo da Velha

**Review by ozhang22 — APPROVED** [review]

**Pull request merged**